README.md

```diff
@@ -3,7 +3,7 @@ zend-diactoros
 
 [![Build Status](https://secure.travis-ci.org/zendframework/zend-diactoros.svg?branch=master)](https://secure.travis-ci.org/zendframework/zend-diactoros)
 
-> Diactoros: an epithet for Hermes, meaning literally, "the messenger."
+> Diactoros: an epithet for Hermes, meaning literally, "the messenger".
```

Comment:
I'm going to revert this.

Proper English placement of final punctuation is *within* the quotes, according to MLA and Strunk & White. Trust me on this.